pkg/controller/perconaservermongodb/status.go

```diff
@@ -66,6 +66,17 @@ func (r *ReconcilePerconaServerMongoDB) updateStatus(cr *api.PerconaServerMongoD
 		}
 	}
 
+	if len(repls) < len(cr.Status.Replsets) {
```

Comment:
what if `len` is the same? I mean, the user "renamed" shard in CR

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/556/commits/00758a20f7e7223e6f23af1eb446b4ba443e9e7b